src/main/java/org/candlepin/insights/orgsync/OrgSyncCompleteEvent.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2009 - 2020 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.insights.orgsync;
+
+
+import org.springframework.context.ApplicationEvent;
+
+/**
+ * Event thrown when an org sync is completed.  Primarily is so the listener can shutdown the pod after
+ * running the job once if the application is deployed to OpenShift as a Cron Job.
+ */
+public class OrgSyncCompleteEvent extends ApplicationEvent {
```

Comment:
Instead of having this be specific to orgSync, I'd prefer we name it something like 'JobCompleteEvent'.